website/addons/wiki/templates/wiki-bar-modal-help.mako

```diff
@@ -0,0 +1,18 @@
+<div class="modal fade" id="wiki-help-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true"></span>&times;</button>
+                    <h3 class="modal-title">Wiki Syntax Help</h3>
+                </div>
+                <div class="modal-body">
+                  <p>
+                    Wiki uses the Markdown syntax. This gives you many options, but can be very simple as well. To see more information and examples go to our <a href="http://help.osf.io/m/collaborating/l/524109-using-the-wiki">guides</a>
```

Comment:
Some nits:

- "Markdown" can link to https://daringfireball.net/projects/markdown/
- Remove extra comma: "This gives you many options but can be very simple as well."
- Add a period after the last sentence.